src/modules/twap/pure/PrimaryActionButton/index.tsx

```diff
@@ -19,22 +17,17 @@ const buttonsMap: Record<TwapFormState, (context: PrimaryActionButtonContext) =>
   ),
   [TwapFormState.NOT_SAFE]: () => (
     <ButtonPrimary disabled={true} buttonSize={ButtonSize.BIG}>
-      Please, connect to Safe
+      Unsupported wallet
```

Comment:
I'm lazy to go up the logic and check this, so I'll just ask:

Will this be displayed also when connected to Safe but via WC?
If not, should we have a state just for that?

Reply:
Ahh now I see it in the description, thank you

Reply:
You will see this (case 4):
![image](https://github.com/cowprotocol/cowswap/assets/7122625/90020fd2-6c31-457e-8cf7-8544d1c22f21)